docs/tfs-server/requirements.md

```diff
@@ -431,7 +431,7 @@ agents on your TFS application tiers.
 
 If you plan to use Code Search, we typically recommend setting up a separate server for it. 
 For more details, see 
-[hardware requirements for Code Search](../search/code/administration.md#hardware-recommendations).
+[hardware requirements for Code Search](../search/code/administration?view=tfs-2017).
```

Comment:
We should have this view as tfs-2018

Reply:
This particular link is from VSTS version of TFS requirements, which by itself is strange. Ideally each version of TFS doc should point to its corresponding Code Search doc.